frontend/components/MainPageV1/Sidebar/Sidebar.tsx

```diff
@@ -129,36 +131,47 @@ const AgentListMenu = ({
   myAgents,
   selectedMenuKeys,
   onAgentSelect,
-}: AgentListMenuProps) => (
-  <Menu
-    selectedKeys={selectedMenuKeys}
-    mode="inline"
-    inlineIndent={4}
-    onClick={onAgentSelect}
-    items={myAgents.map((agent) => ({
-      key: agent.agentType,
-      icon: (
-        <Image
-          src={`/agent-${agent.agentType}-icon.png`}
-          alt={agent.name}
-          width={32}
-          height={32}
-        />
-      ),
-      label: (
-        <Flex justify="space-between" align="center">
-          {agent.name}{' '}
+}: AgentListMenuProps) => {
+  const { selectedService } = useServices();
+  const isAgentRunning =
+    selectedService?.deploymentStatus ===
+    MiddlewareDeploymentStatusMap.DEPLOYED;
+
+  return (
+    <Menu
+      selectedKeys={selectedMenuKeys}
+      mode="inline"
+      inlineIndent={4}
+      onClick={onAgentSelect}
+      items={myAgents.map((agent) => ({
+        key: agent.agentType,
+        icon: (
           <Image
-            src={`/chains/${kebabCase(agent.chainName)}-chain.png`}
-            alt={`${agent.chainName} logo`}
-            width={14}
-            height={14}
+            src={`/agent-${agent.agentType}-icon.png`}
+            alt={agent.name}
+            width={32}
+            height={32}
           />
-        </Flex>
-      ),
-    }))}
-  />
-);
+        ),
+        label: (
+          <Flex justify="space-between" align="center">
+            {agent.name}{' '}
+            {isAgentRunning ? (
+              <AgentRunningAnimation />
+            ) : (
```

Comment:
hmmm will it be shown for all agents if one selected is running? maybe instead of having isAgentRunning outside we should check it like we do in `useAnotherAgentRunning`?